packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -67,6 +73,10 @@ const AccountSwitcher = props => {
         }
     };
 
+    const maxAcountsDisplayed = () => {
```

Comment:
Is this a typo in the function name? Should be accounts?

Reply:
Can we also have an action prefix inside the name like `getMaxAccountsDisplayed`?

Reply:
Oh damn, now I look like an illiterate 😅